packages/babel-types/README.md

```diff
@@ -1076,6 +1076,18 @@ Aliases: `JSX`, `Immutable`
 
 ---
 
+### jSXClosingFragment
+```javascript
+t.jSXClosingFragment()
```

Comment:
I'd actually prefer to name them `jsxClosingFragment`, `jsxElement`, etc, but the pre-existing stuff is already using the weird "SX" prefix so can't help it 😛

Reply:
Separate issue 😛 , yeah whatever is doing that logic needs to handle TS/JSX..